yarn.lock

```diff
@@ -3236,6 +3236,11 @@ ansi-regex@^2.0.0:
   resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-2.1.1.tgz#c3b33ab5ee360d86e0e628f0468ae7ef27d654df"
   integrity sha1-w7M6te42DYbg5ijwRorn7yfWVN8=
 
+ansi-regex@^3.0.0:
```

Comment:
I don't see `package.json` changes, so why does-it changed in `yarn.lock`. Do we know why?

Reply:
@cbourgois it is an issue we have spotted from a moment, we don't have yet found why this package is updated. Sometimes it goes in the other way, deleting this in the yarn.lock file.